### PR TITLE
Product Details Social Media Icons Component Test

### PR DESCRIPTION
**Pull request by kyrianite**
Additionally includes implementation of rough outline of overall product details and linter update attempts

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html>
+<<<<<<< HEAD
```
Comment (davidzny195) [inline client/dist/index.html:3]: This file has some conflicts that aren't deleted

File: client/src/components/ProductDetail/index.jsx [hunk 2]
```diff
@@ -1,11 +1,62 @@
...
+        </div>
+        <div className="product-overview-container">
+          <div className="product-slogan-description">
+            <ProductOverview product={target} />
+          </div>
+          <div className="product-features">
+            <ProductFeatures product={target} />
+          </div>
+        </div>
+        ================================================
```
Comment (davidzny195) [inline client/src/components/ProductDetail/index.jsx:53]: this line is intended or from conflict?

File: client/dist/index.html [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
+  <meta charset="utf-8">
+  <title>FEC</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- <link rel="icon" href="favicon.png"> -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script defer src="bundle.js"></script>
+  <script src="https://kit.fontawesome.com/48000cd12a.js" crossorigin="anonymous"></script>
+</head>
+
+>>>>>>> d9b7c48 (added social media icons)
```
Comment (davidzny195) [inline client/dist/index.html:22]: here

File: client/src/components/QA/index.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const QA = () => (
+  <div>
+    Hello World from QA!
```
[the remark this answers is not in the document]
Comment (kyrianite) [reply client/src/components/QA/index.jsx]: Removed file

File: client/dist/index.html (hunk 3 above)
Comment (kyrianite) [reply client/dist/index.html]: Fixed residual merge issue

File: client/src/components/ProductDetail/index.jsx (hunk 2 above)
Comment (kyrianite) [reply client/src/components/ProductDetail/index.jsx:53]: Intended - just wanted a visual of where my module starts and ends with respect to the other modules

File: client/dist/index.html (hunk 1 above)
Comment (kyrianite) [reply client/dist/index.html]: Fixed

**Review by davidzny195 — APPROVED** [review]

**Pull request merged**